fmriprep/cli/workflow.py

```diff
@@ -150,17 +149,14 @@ def build_workflow(config_file, retval):
     # Check workflow for missing commands
     missing = check_deps(retval['workflow'])
     if missing:
-        build_log.critical(
-            'Cannot run fMRIPrep. Missing dependencies:%s',
-            '\n\t* '.join([''] + [f'{cmd} (Interface: {iface})' for iface, cmd in missing]),
-        )
+        deps_list = '\n\t* '.join([f'{cmd} (Interface: {iface})' for iface, cmd in missing])
+        build_log.critical(f'Cannot run fMRIPrep. Missing dependencies:{deps_list}')
```

Comment:
That was a load-bearing `[''] + `

```suggestion
        deps_list = '\n'.join([f'\t* {cmd} (Interface: {iface})' for iface, cmd in missing])
        build_log.critical(f'Cannot run fMRIPrep. Missing dependencies:\n{deps_list}')
```